src/app/components/Caption/fixture.tsx

```diff
@@ -0,0 +1,174 @@
+import { blockContainingText } from '#models/blocks';
+import { OptimoBlock } from '#app/models/types/optimo';
+
+const generateUUID = () => Math.random().toString(16).slice(2);
+
+export default blockContainingText(
+  'caption',
+  'Some caption text...',
+  'mocked-id',
+);
+
+export const blockContainingPersianText = blockContainingText(
+  'caption',
+  'چیسربرگر',
+  'mocked-id',
+);
+
+export const fragmentBlock = ({
+  text,
+  attributes = [],
+}: {
+  text: string;
+  attributes?: string[];
+}) => ({
+  type: 'fragment',
+  id: generateUUID(),
+  model: {
+    text,
+    attributes,
+  },
+});
+
+export const inlineLinkBlock = ({
+  text,
+  locator,
+  blocks,
+  isExternal,
+}: {
+  text: string;
+  locator: string;
+  blocks: OptimoBlock[];
+  isExternal: boolean;
+}) => ({
+  type: 'urlLink',
+  id: generateUUID(),
+  model: {
+    text,
+    locator,
+    blocks,
+    isExternal,
+  },
+});
+
+export const inlineSpanBlock = ({
+  blocks,
+  language,
+  text,
+}: {
+  text: string;
+  language: string;
+  blocks: OptimoBlock[];
+}) => ({
+  type: 'inline',
+  id: generateUUID(),
+  model: {
+    blocks,
+    language,
+    text,
+  },
+});
+
+const persianText = 'چیسربرگر';
+const persianLink = inlineLinkBlock({
+  text: persianText,
+  locator: 'https://google.com',
+  blocks: [fragmentBlock({ text: persianText })],
+  isExternal: true,
+});
+
+const inlinePersianBlock = inlineSpanBlock({
+  blocks: [persianLink],
+  language: 'fa',
+  text: persianText,
+});
+
+export const blocksWithInline = {
+  model: {
+    blocks: [
+      {
+        model: {
+          blocks: [
+            {
+              model: {
+                blocks: [
+                  fragmentBlock({
+                    text: 'This is some text.',
```

Comment:
Not sure if necessary but the storybook renders without a space between the text and the link which looks a little odd. We could put a space after the full stop to resolve this 

<img width="238" alt="Screenshot 2024-03-07 at 11 52 42" src="https://github.com/bbc/simorgh/assets/98817636/545a8163-afa0-410a-b4b5-27050b2db255">